packages/react/src/evaluation/use-feature-flag.ts

```diff
@@ -70,6 +73,26 @@ T extends boolean
   return query as unknown as ConstrainedFlagQuery<T>;
 }
 
+// alias to the return value of useFlag, used to keep useSuspenseFlag consistent
+type ReturnTypeOfUseFlag<T extends FlagValue> = ReturnType<typeof useFlag<T>>
```

Comment:
Aliased this type so `useFlag` and `useSuspense` flag stay synced up.

Reply:
Just a naming thing but I would expect a type name like `UseFlagReturn` or even `UseFlagResult`.